adoc/chapters/programming_interface.adoc

```diff
@@ -12501,7 +12503,7 @@ sycl::logical_and
 a@
 [source]
 ----
-std::is_same_v<std::remove_v_t<AccumulatorT>, bool>
+std::is_same_v<std::remove_cv_t<AccumulatorT>, bool>
```

Comment:
This fix is already made in #352.  It's better to remove yours to avoid merge conflict.